### ISSUE TITLE
About satellite image resolution

### ISSUE THREAD

**Issue by lbhwyy** [yzdad/D-GLSNet#5]
Thank you for your amazing open-source work! I have a question about the satellite images I downloaded using the links provided by your team(https://gac-geo.googlecnapps.cn/maps/vt?lyrs=s&x=68590&y=45022&z=17). The images I obtained through the links are 256x256 in resolution, but the satellite images from the dataset shared by you are 1024x1024. I'm curious to know how you acquired such high-resolution satellite images.

**Comment by yzdad** [comment]
Tiles can only be downloaded one by one, and each piece is 256 * 256. If a higher resolution is required, then all the files in this area need to be downloaded and concatenated.

**Comment by lbhwyy** [comment]
OK, is the zoom level of your satellite image 19?

**Comment by yzdad** [comment]
Yes, the zoom level of the satellite image we obtained is 19, but we reduced it by eight times in subsequent processing.
